web-common/src/components/forms/DatePicker.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  export let value: string;
+  export let id: string;
+  export let label: string;
+</script>
+
+<div class="flex flex-col gap-y-2">
+  <label for={id} class="text-gray-800 text-sm font-medium">{label}</label>
+  <!-- Get calendar day -->
+  <input
+    type="date"
+    class="bg-white rounded-sm border border-gray-300 px-3 py-[5px] h-8 cursor-pointer focus:outline-blue-500 w-full text-xs"
+    bind:value
+  />
+</div>
```

Comment:
Any we not using the custom date picker we have in our time range?

Reply:
Yeah, I'd like to, but our current `LitePicker` interface only supports selecting date ranges, not individual dates. This component was a quick stop-gap solution.

Anyways, I'm not actually using this component anywhere yet, so I've just removed it for now.